scripts/ci

```diff
@@ -3,10 +3,10 @@ set -e
 
 cd $(dirname $0)
 
-./build
-./test
 ./validate
 ./validate-ci
+./build
```

Comment:
This change looks like it's breaking GHA, although I don't see why.

Reply:
Agreed, reverted. We can look at this outside of this PR.